src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -84,6 +84,14 @@
     "In the current release": "resolved:inCurrentRelease",
 }
 
+ARCHIVE_OPTIONS = {
+    "Until escalating": "ignored:until_escalating",
+    "Until 10 events": "ignored:until_condition_met:10",
+    "Until 100 events": "ignored:until_condition_met:100",
+    "Until 1000 events": "ignored:until_condition_met:1000",
+    "Forever": "ignored:forever",
```

Comment:
i realize that these versions of the substatus options are deprecated (see https://github.com/getsentry/sentry/blob/master/src/sentry/types/group.py#L42); should i use the ones that aren't deprecated (would include changing `until_escalating` which we've used even before block kit changes)?

Reply:
I think that makes sense to do, it looks like `until_escalating` and `archived_until_escalating` are the same anyway